### DISCUSSION TITLE
Startup configuration screen

### DISCUSSION THREAD

**Discussion by GitGianluc** [URUWorks/TeroSubtitler#93] [Q&A] [answered]
Is it possible to access the settings screen that is displayed on the first launch (ffmpeg / whisper.cpp configuration, etc.) again? It would be helpful to change or check the initially set configurations, but I don't see any way to return to that menu after the first launch.

**Comment by chenlung** [comment]
I think a section called Dependencies in Settings would be good. Also useful in case new versions of them are released

**Reply by GitGianluc** [reply]
OK, but currently there is no way to access that settings screen again after the initial launch. Can you confirm that?

**Reply by URUWorks** [reply, marked as answer]
For now, the only way to get the wizard to reappear is by deleting the tero.cfg file, but obviously you lose all the previous configuration.
But if it seems useful to you, we can change this.

**Reply by chenlung** [reply]
Another advantage would be choosing the version of a dependency, whether it's custom-built or because an older one is needed (i.e. newer won't run or has issues — namely mpv on older macOS).

FFmpeg in Settings can be relocated here as well.

**Reply by GitGianluc** [reply]
> But if it seems useful to you, we can change this.

Yes, please!
